app/src/main/java/hr/foi/air/servicesync/ui/components/BackButton.kt

```diff
@@ -0,0 +1,26 @@
+package hr.foi.air.servicesync.ui.components
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun BackButton(onBackPressed : () ->Unit, modifier: Modifier = Modifier) {
```

Comment:
I must this is very well done in terms of reusability of the code. Thanks!

Reply:
No problem!